docs/tracing/create-tracing-app.md

```diff
@@ -0,0 +1,254 @@
+- Methods: Helm, kubectl, Sematext Operator
+- Deployment: As a DaemonSet across your Kubernetes cluster
+
+3.2 Follow Installation Instructions:
+
+The UI provides step-by-step installation commands for your chosen method:
+
+Agent Configuration:
+The agent will be automatically configured with:
+
+- Your Infra App token for authentication
+- OTLP receiver enabled for trace collection  
+- Your Traces token for sending data to Sematext Cloud
+- Service names you configured in Step 2
+- Default ports: 4337 (gRPC), 4338 (HTTP)
+
+Custom Port Configuration (Optional):
+
+- Modify default OTLP ports if needed
+
+### Step 4: Complete Setup
+
+4.1 Verify Installation:
+
+- The wizard will show "Host Detected" when the agent is running correctly
+- Services will appear as "Healthy" when properly configured
+
+4.2 Generate Test Traffic:
+
+- Make requests to your instrumented application
+- Traces should appear in the newly created Tracing App within seconds
+
+## What You'll Have After Completion
+
+### Tokens and Configuration
+
+- Infra App Token: For agent authentication
+- Traces Token: For sending trace data to Sematext Cloud
+- Service Configuration: All configured services ready to send traces
+
+### Ready-to-Use Features
+
+- Tracing Overview: Key metrics and service health
+- Traces Explorer: Search and filter traces
+- Individual Trace Analysis: Detailed span inspection
+- Alert Configuration: Ready to set up performance alerts
+
+## SDK Examples
+
+Here are quick examples for popular languages:
+
+### Java
+```bash
+# Download OpenTelemetry Java agent
+wget https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar
+
+# Run with automatic instrumentation
+java -javaagent:opentelemetry-javaagent.jar \
+  -Dotel.service.name=your-service-name \
+  -Dotel.exporter.otlp.endpoint=http://localhost:4338 \
+  -Dotel.exporter.otlp.protocol=http/protobuf \
+  -jar your-application.jar
+```
+
+### Python
+```bash
+# Install OpenTelemetry packages
+pip install opentelemetry-distro[otlp]
+opentelemetry-bootstrap -a install
+
+# Configure and run
+export OTEL_SERVICE_NAME=your-service-name
+export OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4338
+export OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf
+
+opentelemetry-instrument python your_app.py
+```
+
+### Node.js
+```bash
+# Install dependencies
+npm install @opentelemetry/auto-instrumentations-node @opentelemetry/exporter-trace-otlp-http @opentelemetry/sdk-node
+
+# Create tracing.js and run
+node -r ./tracing.js your-app.js
+```
+
+For complete setup instructions, the wizard provides detailed, language-specific guidance.
+
+## Best Practices
+
+### Naming Conventions
+
+- Apps: Use environment prefixes (`prod-api`, `staging-web`)
+- Services: Use descriptive names (`user-authentication`, `payment-processor`)
+- Keep names concise but clear
+
+### Planning Your Setup
+
+- One App per Service: Separate Tracing Apps for different microservices
```

Comment:
Hm, is this really true? Wouldn't you want 1 Tracing App for your whole "business application", which may contain one or N services? If you have different microservices in different Apps and these microservices communicate, you wouldn't be able to see that, would you? Or it would be awkward because it wouldn't be in the same App, no?

Reply:
100%